LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -5,6 +5,7 @@
     "css/css-animations/parsing": "import",
     "css/css-box": "import",
     "css/css-color": "import",
+    "css/css-counter-styles/cssom/": "import",
```

Comment:
please undo this change, it's already covered below in this file with `"web-platform-tests/css/css-counter-styles": "import",`